index.html

```diff
@@ -47,6 +45,186 @@ <h5>LET'S CONNECT</h5>
         </ul>
       </div>
     </section>
+<!-- NEW FROM HERE -->    
+    <section class="work-grid-container">
+      <section class="work1">
+        <img src="./images/Snapshoot.png" alt="Topic Snapshoot">
+        <div id="block1">
+          <h2>Tonic</h2>
+          <div id="work1frame">
+            <ul>
+              <li><div id="work1text1"><h5>CANOPY</h5></div></li>
+              <li><div id="work1counter"></div></li>
+              <li><div id="work1role"><h5>Back End Dev</h5></div></li>
+              <li><div id="work1counter1"></div></li>
+              <li><div id="work1text"><h5>2015</h5></div></li>
+            </ul>
+          </div>
+          <div class="work1primary-text">
+            <p>A daily selection of privately personalized reads; no accounts or 
+            sign-ups required.</p>
+          </div>
+          <div id="work1tags">
+            <ul>
+              <li><div id="work1tag1"><div id="work1html1">html</div></div></li>
+              <li><div id="work1tag2"><div id="work1css1">css</div></div></li>
+              <li><div id="work1tag3"><div id="work1javascript1">javascript</div></div></li>
+            </ul>                        
+          </div>
+          <div class="work1Actions">
+            <button type="button" id="work1see-project">
+              <strong>See Project</strong>
+            </button>
```

Comment:
- [ ] [OPTIONAL] Great job making the requested change on the buttons. Kindly check your buttons, for some reason, they are not fitting in the container of the button.

**This is how I am currently seeing the buttons:**

![image](https://user-images.githubusercontent.com/86318284/185759739-c96e2b3c-0985-48d0-b0ce-cbea7ce45cae.png)

**Kindly make this change in all the buttons as I have realized that the trend is similar**

Reply:
@Nemwel-Boniface in the last check that I perform the button's text rendered properly. I'll make sure to make the appropriate changes. Thank you very much for pointing that out!